Chindea Tudor Eugen/Lab3 CP/main.c

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
+{
+    // continue - este folosit in for  si cand se apeleaza se intoarce la inceput de for si creste contorul
+    // continue e poate folosi in orice functie repetitiva; te aduce la inceputul instructiunilor
+    int sequence[100],n,mini,pozmin[100],maxi,pozmax[100],ctmin=1,ctmax=1,i;
+    scanf("%d",&n);
+    for(i=0; i<n; i++)
+    {
+        scanf("%d",&sequence[i]);
+        if(i==0)
```

Comment:
You should initialize `mini`,`maxi`,`pozmin` and `pozmax` at the beginning so you won't need to use this if here
Also, why are `pozmin` and `pozmax` vectors? it doesn't make sense

Reply:
Fair enough, you are right on that